scripts/gha/summarize_test_results.py

```diff
@@ -360,15 +361,15 @@ def reorganize_configs(configs):
 #     ['ubuntu', 'windows']
 #     -> ['2/3 os: ubuntu,windows': ]
 def combine_configs(error_configs, all_configs):
-  logging.info("error_configs: %s; all_configs: %s", error_configs, all_configs)
+  # logging.info("error_configs: %s; all_configs: %s", error_configs, all_configs)
```

Comment:
do we still need those? And the one below?

Reply:
No. This is only for debugging purposes.